transcriptic/analysis/kinetics.py

```diff
@@ -123,8 +124,8 @@ def plot(self, wells="*", groupby=None, title=None, xlabel=None, ylabel=None, ma
                                  name=self.properties["name"].loc[well]) for well in wells]
         else:
             if groupby not in self.properties.columns:
-                raise ValueError("\'%s\' not found in the properties table. Please specify a column which exists" %
-                                 groupby)
+                raise ValueError(f"'{groupby}' not found in the properties table. "
+                                 f"Please specify a column which exists")
```

Comment:
Same here. 
f"'{groupby}' not found in the properties table. Please specify a column which exists"